src/components/MenuItems.tsx

```diff
@@ -54,57 +54,69 @@ const SettingsMenuListItem = () => {
 
```

Comment:
**Note:**

GitHub isn't showing the changes I've made in this file very well.
**All I've done here is wrap tooltips around menu items.** And ensured the `Tooltip` text matches the `ListItemText`

Reply:
There's a setting that allows you to ignore whitespace changes when reviewing a PR. It's quite handy for when blocks of text have their indentation changed: 
![image](https://user-images.githubusercontent.com/2938284/124495723-5f6e9f80-ddb0-11eb-9adc-74676ff70a9a.png)

Reply:
Oh brilliant, that's great! Cheers for pointing that out